dotnet/src/extensions/Azure/Handlers/Helpers/FunctionMetadata.cs

```diff
@@ -0,0 +1,134 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace GeneXus.Deploy.AzureFunctions.Handlers.Helpers
+{
+	public class FunctionMetadata
+		{
+			public const string directionIn = "In";
+			public const string directionOut = "Out";
+			public class TriggerTypes
+			{
+				public const string TimerTrigger = "timerTrigger";
+				public const string QueueTrigger = "queueTrigger";
+				public const string ServiceBusTrigger = "serviceBusTrigger";
+				public const string HttpTrigger = "HttpTrigger";
+				public const string Http = "http";
+			}
+			public string name { get; set; }
+			public string scriptFile { get; set; }
+			public string entryPoint { get; set; }
+			public string language { get; set; }
```

Comment:
In this case it's because they are serialized as JSON into a configuration file with that name. I could have used [JsonProperty("entryPoint")] for example. I'll consider it next time.